### PR TITLE
Feature/atom input

### PR DESCRIPTION
**Pull request by dannil76**

### CODE REVIEW

File: src/components/atoms/Input.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  width: 100%;
+`;
+
+const TextInput = () => <input type="text" />;
+
+const Input = styled(TextInput)`
+  ${input}
+`;
+
+Input.propTypes = {};
```
Comment (TeddyAndersson) [inline src/components/atoms/Input.js]: All props in defaultProps should be listed in propTypes as not required.

File: src/components/atoms/Button.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
+/* eslint-disable no-plusplus */
```
Comment (TeddyAndersson) [inline src/components/atoms/Button.js]: Check if there are problems with the current button component in develop branch

File: src/components/atoms/Input.stories.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+//     {...props}
+//   />
+// ));
+
+export default {
+  title: 'Input',
+  component: Input,
+};
+
+export const Default = () => (
```
Comment (TeddyAndersson) [inline src/components/atoms/Input.stories.js]: Om du orkar så kommentera gärna bort de Stories som just nu är tomma :D

File: src/components/atoms/Button.js (hunk 2 above)
Comment (TeddyAndersson) [reply src/components/atoms/Button.js]: Would be best if you could remove the button commit from this PR, feels like it have just hitchhiked these changes for some reason :P

File: src/components/atoms/Input.stories.js (hunk 3 above)
Comment (dannil76) [reply src/components/atoms/Input.stories.js]: Done!

**Review by nRamstedt — APPROVED** [review]

**Pull request merged**